android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -372,6 +389,67 @@ public void finishNativeInitialization() {
                     calender.getTimeInMillis());
         }
         checkSetDefaultBrowserModal();
+        if (mBraveRewardsNativeWorker != null) {
+            mBraveRewardsNativeWorker.StartProcess();
```

Comment:
Maybe we shouldn't start it where rewards are not available, x86 devices?

Reply:
Good point. updating it.